src/main/java/com/github/wasiqb/coteafs/selenium/core/base/driver/CustomScreenRecorder.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2019, Wasiq Bhamla.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.wasiqb.coteafs.selenium.core.base.driver;
+
+import static java.text.MessageFormat.format;
+import static org.monte.media.FormatKeys.EncodingKey;
+import static org.monte.media.FormatKeys.FrameRateKey;
+import static org.monte.media.FormatKeys.KeyFrameIntervalKey;
+import static org.monte.media.FormatKeys.MIME_AVI;
+import static org.monte.media.FormatKeys.MediaTypeKey;
+import static org.monte.media.FormatKeys.MimeTypeKey;
+import static org.monte.media.Registry.getInstance;
+import static org.monte.media.VideoFormatKeys.CompressorNameKey;
+import static org.monte.media.VideoFormatKeys.DepthKey;
+import static org.monte.media.VideoFormatKeys.ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE;
+import static org.monte.media.VideoFormatKeys.QualityKey;
+
+import java.awt.AWTException;
+import java.awt.Dimension;
+import java.awt.GraphicsConfiguration;
+import java.awt.GraphicsEnvironment;
+import java.awt.Rectangle;
+import java.awt.Toolkit;
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import com.github.wasiqb.coteafs.selenium.config.ConfigUtil;
+import com.github.wasiqb.coteafs.selenium.config.RecorderSetting;
+
+import org.monte.media.Format;
+import org.monte.media.FormatKeys.MediaType;
+import org.monte.media.math.Rational;
+import org.monte.screenrecorder.ScreenRecorder;
+
+/**
+ * @author Wasiq Bhamla
+ * @since 27-Oct-2019
+ */
+class CustomScreenRecorder extends ScreenRecorder {
+    private static final RecorderSetting RECORDER_SETTING = ConfigUtil.appSetting ()
+        .getPlayback ()
+        .getRecording ();
+    private static ScreenRecorder        screenRecorder;
+
+    static void startRecording () throws Exception {
```

Comment:
Appropriate logs should be written in console/file when video recording starts.
Also, logs should be displayed stating "video recording is set to off" when video recording is not enabled.